packages/datatrak-web/src/views/AccountSettingsPage/PersonalDetailsSection.tsx

```diff
@@ -4,10 +4,168 @@
  */
 
 import React from 'react';
+import styled from 'styled-components';
+import { useForm, Controller, SubmitHandler } from 'react-hook-form';
 import { AccountSettingsSection } from './AccountSettingsSection';
+import { Button } from '../../components';
+import { successToast } from '../../utils';
+import { TextField } from '@tupaia/ui-components';
+import { useCurrentUser } from '../../api';
+import { useEditUser } from '../../api/mutations';
+import { UserAccountDetails } from '../../types';
```

Comment:
```suggestion
import styled from 'styled-components';
import { useForm, Controller, SubmitHandler } from 'react-hook-form';
import { TextField } from '@tupaia/ui-components';
import { AccountSettingsSection } from './AccountSettingsSection';
import { Button } from '../../components';
import { successToast } from '../../utils';
import { useCurrentUser, useEditUser } from '../../api';
import { UserAccountDetails } from '../../types';
```

Reply:
there is a lint rule that specifies that imports outside of the package should come before ones inside of it

Reply:
Oops, should I be enforcing this manually, or have I misconfigured my editor?